### DISCUSSION TITLE
Modify about: button to open page in popup rather than new tab

### DISCUSSION THREAD

**Discussion by emvaized** [Aris-t2/CustomJSforFx#93] [Q&A]
Hi there! 
Is it possible to modify [Generic about button](https://github.com/Aris-t2/CustomJSforFx/blob/master/scripts/about_button_generic.uc.js) in such way, that on click it will open page in a popup, rather than in a new tab? To make it behave like Firefox's built-in History button or Downloads button. 
Thanks!

PS: Sorry if my question sounds dumb, I'm quite new to this whole userChrome.js topic and not aware yet about its capabilities and technical limitations.

**Comment by Aris-t2** [comment]
That is possible.

Replace
`win.gBrowser.selectedTab = win.gBrowser.addTrustedTab('about:about');`

with
`window.open("about:about","","width=800,height=600, chrome");`

**Reply by Speravir** [reply]
Aris, how about making this an option/setting?

**Reply by Aris-t2** [reply]
Done.
https://github.com/Aris-t2/CustomJSforFx/blob/master/scripts/about_button_generic.uc.js
